examples/sycl/00_pvc_gemm/00_pvc_gemm.cpp

```diff
@@ -82,6 +82,11 @@ using namespace cute;
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
+// The alignment requirement in bytes on inner dimmension that will work for both PVC and BMG
```

Comment:
Sorry for a slightly cumbersome late suggestion - this didn't occur to me on my first review, but I wonder if we want to make this a dedicated example? Or, at least, apply it to an example other than `00_pvc_gemm`. Given this will be many developers first intro to cutlass-sycl, it would be nice not to hit them in the face with some potentially daunting alignment & padding calculations straight away.

Reply:
Maybe, but I am unsure if this is a big enough change for its own example. Also, alternatively we hit them with many inputs being unsupported. I am not sure. I would prefer to hear other opinions on this.

Reply:
split into separate example